### PR TITLE
chore: update playwright and setup draft in github bot for future test filtering

### PR DESCRIPTION
**Pull request by valpinkman**
### 📝 Description

Update Playwright to latest version
Prepare github bot to send status of PR to jobs (useful to run a specific test suite based on the PR status)

### ❓ Context

- **Impacted projects**: `automation`, `lld` 
- **Linked resource(s)**: `LIVE-7744` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .github/workflows/test-desktop-external.yml [hunk 1]
```diff
@@ -82,7 +87,12 @@ jobs:
           install_playwright: true
       - name: Run playwright tests [Linux => xvfb-run]
         id: tests
-        run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- pnpm desktop test:playwright
+        run: |
+          if [[ ${{ inputs.draft }} == true ]]; then
+            xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- pnpm desktop test:playwright
```
Comment (ggilchrist-ledger) [inline .github/workflows/test-desktop-external.yml:92]: What's this if statement for?

Comment (ggilchrist-ledger) [reply .github/workflows/test-desktop-external.yml:92]: Is it just preparation for something else to come regarding tagging?

**Comment by ggilchrist-ledger** [comment]
LGTM. I'm assuming the `if` statements in the workflows will be different once the actual test suite is updated to run smoke/full tests. If not then let me know what they are for :)

**Review by ggilchrist-ledger — APPROVED** [review]

**Comment by valpinkman** [comment]
> LGTM. I'm assuming the `if` statements in the workflows will be different once the actual test suite is updated to run smoke/full tests. If not then let me know what they are for :)

Exactly :)

**Pull request merged**